account_invoice_force_removal/models/account_move.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    types_allow_unlink = ["out_invoice", "out_refund", "in_invoice", "in_refund"]
+
+    def unlink(self):
+        """Allow to remove moves related to invoices."""
+        moves = self.filtered(lambda m: m.type in self.types_allow_unlink).filtered(
```

Comment:
Don't do this `filtered`, but directly call `super` with the context. Or the rest of the checks (like the posted one) is ignored also by the context?

Reply:
@pedrobaeza the unique check is this:
https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1741-L1742

Reply:
I see, then it doesn't serve as is, as this will allow to unlink posted entries. I will comment in the other PR. Sorry for the noise.